<launch>
   <!-- comentarios -->
    <arg name="model" />
	<arg name="gui" default="true" />
	<param name="robot_description" textfile="$(find my_delta_robot)/urdf/delta_robot.urdf" />
	<param name="use_gui" value="$(arg gui)"/>
	<node name="robot_pub" pkg="robot_state_publisher" type="robot_state_publisher" />
	<node name="rviz" pkg="rviz" type="rviz" args="âˆ’d $(find my_delta_robot)/rviz/my_config.rviz" required="true" />

	<node name="main_node" pkg="my_delta_robot" type="main_node" output="screen"/>
	<!-- <node name="node_b" pkg="my_delta_robot" type="node_b" output="screen"/>
	<node name="node_a" pkg="my_delta_robot" type="node_a" output="screen"/>
	<node name="serial_module" pkg="my_delta_robot" type="serial_module" /> -->
</launch>
